### DISCUSSION TITLE
Can Horovod support two optimizers in one model training?

### DISCUSSION THREAD

**Discussion by gongheguoyingpai** [horovod/horovod#2848] [Q&A]
I have a model use bert as my pretrained model, and add some mlp layers。I want to use two optimizers in training, one optimizer use a low learning rate to optimize bert params, the other optimizer use a high learning rate to optimize mlp params。I want to know, can horovod support two or more optimizers in one model training? 

Does anyone know? Thanks!
